libraries/matrix/impl/src/test/kotlin/io/element/android/libraries/matrix/impl/room/preview/RoomPreviewMapperTest.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Please see LICENSE in the repository root for full details.
+ */
+
+package io.element.android.libraries.matrix.impl.room.preview
+
+import com.google.common.truth.Truth.assertThat
+import io.element.android.libraries.matrix.api.room.RoomType
+import io.element.android.libraries.matrix.api.room.preview.RoomPreview
+import io.element.android.libraries.matrix.test.A_ROOM_ALIAS
+import io.element.android.libraries.matrix.test.A_ROOM_ID
+import org.junit.Test
+import org.matrix.rustcomponents.sdk.RoomPreview as RustRoomPreview
+
+class RoomPreviewMapperTest {
+    @Test
+    fun `map should map values 1`() {
```

Comment:
Kind of a generic test name 😅 .

Reply:
Yeah, I tried not to spend hours looking for perfect names for tests like this :)